test/parallel/test-assert-deep.js

```diff
@@ -1220,3 +1220,60 @@ assert.throws(
 
   assertNotDeepOrStrict(a, b);
 }
+
+// eslint-disable-next-line node-core/crypto-check
+if (common.hasCrypto) {
```

Comment:
nit: if you invert the test, the linter no longer complains
```suggestion
if (!common.hasCrypto)
  return;
```